src/app/modules/group/components/group-log-view/group-log-view.component.ts

```diff
@@ -19,22 +31,34 @@ interface Data {
   templateUrl: './group-log-view.component.html',
   styleUrls: [ './group-log-view.component.scss' ],
 })
-export class GroupLogViewComponent implements OnChanges, OnDestroy {
+export class GroupLogViewComponent implements OnChanges, OnInit, OnDestroy {
 
   @Input() groupId?: string;
   @Input() showUserColumn = true;
 
+  @ViewChild('op') op?: OverlayPanel;
+  @ViewChildren('contentRef') contentRef?: QueryList<ElementRef<HTMLElement>>;
+
+  private readonly unsubscribe$ = new Subject<void>();
   private readonly groupId$ = new ReplaySubject<string>(1);
   private readonly refresh$ = new Subject<void>();
   readonly state$ = this.groupId$.pipe(
     switchMap((groupId: string) => this.getData$(groupId)),
     mapToFetchState({ resetter: this.refresh$ }),
   );
+  private readonly showOverlaySubject$ = new BehaviorSubject<{ event: Event, itemId: string, target: HTMLElement }|undefined>(undefined);
+  showOverlay$ = this.showOverlaySubject$.asObservable().pipe(debounceTime(750), shareReplay(1));
 
   constructor(
     private activityLogService: ActivityLogService,
   ) {}
 
+  ngOnInit(): void {
+    this.showOverlay$.pipe(filter(isNotUndefined), takeUntil(this.unsubscribe$)).subscribe(data =>
+      this.op?.toggle(data.event, data.target)
+    );
+  }
+
```

Comment:
Sorry I don't know where it went, I actually added a comment previously on these lines... :confused: 

Here `.pipe(..., takeUntil(unsubscribe$).subscribe(...)` looks like a subscription you want to unsubscribe from on delete. For that you can directly store the subscription and unsubscribe on delete:
```ts
private showOverlaySubscription = this.showOverlay$.pipe(filter(...)).subscribe(...);

ngOnDelete() {
  ...
  this.showOverlaySubscription.unsubscribe();
}
```

And you can remove the property `unsubscribe$` :grin:

Reply:
Fixed